release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -1923,6 +1906,24 @@ In this update, `systemd` service only sets the default RPS mask for virtual int
+
+* Previously, the topology view in the {product-title} web console would fail or lag for clusters with more than 100 nodes. With this update, the topology view shows a `LimitExceeded` state for clusters with more than 100 nodes. An option is provided to view the resources by using the *Search* page instead. Alternatively, you can click Show topology anyway to continue to load the topology view. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2060329[*BZ#2060329*])
+
+* Previously, if a service exposed multiple service ports and the route target port was `8080`, attempts to change the target port would result in another service port being updated instead of the port `8080` service port. With this update, the service port corresponding to the active target port is replaced when a new target port is set. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2077943[*BZ#2077943*])
+
+* Previously, the `git` detection used to manage instance APIs to get repository information did not work for repositories from self hosted GitHub and Bitbucket. With this update, detection for self hosted GitHub and Bitbucket instance repositories works. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2038244[*BZ#2038244*])
+
+* Previously, `apiVersion` was not passed in the correct format to *Resource* drop-down menu in the *EventSource* creation form. This caused `InContext` to not be selected under *EventSource* creation, which excluded it from the *Resource* drop-down menu. With this update, the *Resource* drop-down menu includes `Resource` from `InContext`. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2070020[*BZ#2070020*])
+
+* Previously, the *Pipeline metrics* page displayed all API calls for the metrics query and failed with a `404` error. With this update, the `prometheus-tenancy` API is used to get the metrics data for the pipeline. Now, the pipeline metrics page displays all data and graphs to the non-admin user with at least view access to the namespace. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2041769[*BZ#2041769*])
+
+* Previously, you could access a quick search and add modal with the Crtl+space keyboard shortcut, but you could not close them by using the same keyboard shortcut. With this update, you can close the quick search and add modal using the Ctrl+space keyboard shortcut. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2093586[*BZ#2093586*])
+
+* Previously,  the resources created for user settings were not removed if the user is deleted. Consequently, the created resources in the `openshift-console-user-settings` namespace was never removed. With this update, an `ownerReference` is added to your metadata at creation. This allows automatic removal of the resources when the user no longer exists. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2019564[*BZ#2019564*])
```

Comment:
You could change "if the user is deleted" to "if the user was deleted" in the first line.